quipucords/scanner/vcenter/inspect.py

```diff
@@ -199,13 +201,17 @@ def parse_vm_props(self, props, host_dict):
             elif prop.name == 'summary.config.uuid':
                 facts['vm.uuid'] = prop.val
             elif prop.name == 'runtime.host':
-                host_facts = host_dict[str(prop.val)]
-                facts['vm.host.name'] = host_facts['host.name']
-                facts['vm.host.cpu_cores'] = host_facts['host.cpu_cores']
-                facts['vm.host.cpu_count'] = host_facts['host.cpu_count']
-                facts['vm.host.cpu_threads'] = host_facts['host.cpu_threads']
-                facts['vm.cluster'] = host_facts['host.cluster']
-                facts['vm.datacenter'] = host_facts['host.datacenter']
+                if host_dict.get(str(prop.val)):
+                    host_facts = host_dict[str(prop.val)]
```

Comment:
we should do `host_dict.get(str(prop.val))` here too

Reply:
if the get on line 204 resolved then accessing the key directly should be fine here. I've suggested another way on my review which avoids calling get two times.

Reply:
I checked in the condition above if it was in the keys before accessing it. However, I can change it as well.

Reply:
Nice! no worries changing it here then!